tools/provision

```diff
@@ -32,17 +32,14 @@ the Python version this command is executed with."""
     options = parser.parse_args()
 
     base_dir = os.path.abspath(os.path.join(__file__, '..', '..'))
-    py_version_output = subprocess.check_output([options.python_interpreter, '--version'],
-                                                stderr=subprocess.STDOUT, universal_newlines=True)
-    # The output has the format "Python 1.2.3"
-    py_version_list = py_version_output.split()[1].split('.')
-    py_version = tuple(int(num) for num in py_version_list)
+
+    py_version = tuple(sys.version_info)  # (3, 6, 7, 'candidate', 1) for Python 3.6.7rc1
```

Comment:
I don't think this change is correct, since you're now ignoring the `-p` option.  We want 
`./tools/provision -p /usr/bin/python2.7` to throw this error.  

Instead, we should just fix the parsing logic to handle `rc1` type values.

Reply:
(I think a reasonable option would be to just put `[0:2]` at the end of the old `py_version_list` definition, since we don't need the minor version at all in the following code).